go.mod

```diff
@@ -401,7 +401,7 @@ replace github.com/hashicorp/go-hclog => github.com/hashicorp/go-hclog v0.16.1
 // This is a patched v0.8.2 intended to fix session.Find (and others) silently ignoring SQLITE_BUSY errors. This could
 // happen, for example, during a read when the sqlite db is under heavy write load.
 // This patch cherry picks compatible fixes from upstream xorm PR#1998 and can be reverted on upgrade to xorm v1.2.0+.
-replace xorm.io/xorm => github.com/grafana/xorm v0.8.3-0.20220614223926-2fcda7565af6
```

Comment:
I would leave this replace line as a comment, so that our team could comment/uncomment it easily, but the rest of the company, releases and the CI used the normal xorm

Reply:
Sorry, I meant the other way around: keep the original github.com/grafana/xorm for now and add a commented-out local pkg/util/xorm so that it would be easier for our team to temporarily uncomment and experiment with it.